install_config/install/rpm_vs_containerized.adoc

```diff
@@ -145,27 +144,17 @@ also provided to ease administrative tasks:
 |Kubernetes CLI
 |===
 
-The wrapper spawns a new container on each invocation, so you may notice
-it run slightly slower than native CLI operations.
-
-The wrapper scripts mount a limited subset of paths:
-
-- *_~/.kube_*
-- *_/etc/origin/_*
-- *_/tmp/_*
-
-Be mindful of this when passing in files to be processed by the `oc` or `oadm`
-commands. You may find it easier to redirect the input, for example:
+Be mindful of these links when you pass in files to be processed by the `oc` or
+`oadm` commands. You might find it easier to redirect the input, for example:
```

Comment:
We have started deprecating the use of oadm command. So might be good opportunity to remove it here.